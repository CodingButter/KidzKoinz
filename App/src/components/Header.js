import React from "react";
import styled from "styled-components";
import Edges from "./Edges";

import LogoImage from "../img/logo.png";

export default props => {
  return (
    <Header>
      <Edges>
        <Logo src={LogoImage} />
      </Edges>
    </Header>
  );
};

const Header = styled.div`
  display: flex;
  justify-content: flex-start;
  align-items: center;

  background: linear-gradient(
    90deg,
    rgba(255, 255, 255, 1) 0%,
    rgba(255, 255, 255, 0) 70%
  );

  padding: 10px 20px;
  height: 110px;
`;

const Logo = styled.img`
  width: 110px;
`;
