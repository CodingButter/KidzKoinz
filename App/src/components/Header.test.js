import Header from "./Header";
describe("shallow", () => {
  it("Sould return the logo an nav", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
