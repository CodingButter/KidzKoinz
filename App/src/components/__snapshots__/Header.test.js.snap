// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow Sould return the logo an nav 1`] = `
<styled.div>
  <_default>
    <Link
      to="/"
    >
      <styled.img
        src="logo.png"
      />
    </Link>
    <styled.nav>
      <Link
        to="/login"
      >
        Login
      </Link>
    </styled.nav>
  </_default>
</styled.div>
`;
